stl/inc/chrono

```diff
@@ -1331,15 +1331,15 @@ namespace chrono {
         return *this;
     }
 
-    inline constexpr day _Last_day_table[] = {
-        day{31}, day{28}, day{31}, day{30}, day{31}, day{30}, day{31}, day{31}, day{30}, day{31}, day{30}, day{31}};
+    inline constexpr day _Last_day_table[] = {day{31}, day{28}, day{31}, day{30}, day{31}, day{30}, day{31}, day{31},
+        day{30}, day{31}, day{30}, day{31}, day{42}, day{42}, day{42}, day{42}};

```

Comment:
42 is confusing. Maybe just 0?

Reply:
I didn't want to just outright disrespect Casey like that.

Reply:
Now we can argue about the precise value: I am firmly of the belief that it should be `day{0}` or greater than `day{31}` so it's outside of the `ok()` range. I am slightly less certain that it should be a large enough number to create high relative error in calculations so bugs are more readily apparent. You may not notice something is off by 1 or 2 until shipping, but hopefully a calculation that requires a month with 137 days to make sense will catch your eye.

Given the above, I'm inclined to suggest a number like `day{45}` or `day{65}` that's precisely between an integral number of months worth of days. Despite being largely a joke in the issue discussion, `day{42}` wouldn't seem to be horrible.

Opinions?

Reply:
Oh, and whatever magic number we land upon, we should have a clarifying comment here. Otherwise, the PR looks good to me.

Reply:
I'd consider either `day{0}` or `day{255}` as the extreme values, but ultimately I'd recommend `day{255}`. It will immediately appear to be garbage in a way that `day{0}` might not always be (imagine code that is transforming these values back to integers, and summing them up; adding `0` might go unnoticed, but adding `255` will look bogus way faster). As an aside, 255 / 30 = 8.5 exactly.